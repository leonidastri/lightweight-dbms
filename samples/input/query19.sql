select * from Sailors, Boats, Reserves where Sailors.A = Reserves.G AND Boats.D > Reserves.H AND Sailors.A = Boats.E;