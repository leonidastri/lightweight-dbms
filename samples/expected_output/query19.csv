1,200,50,103,1,1,1,101
1,200,50,103,1,1,1,102
2,200,200,107,2,8,2,101